setup.py

```diff
@@ -29,6 +29,6 @@
     license="GPL-3.0",
     packages=find_packages(exclude=["tests", "tests.*"]),
     install_requires=REQUIRES,
-    python_requires=">=3.7",
+    python_requires=">=3.7,<3.11",
```

Comment:
Is this necessary?

Reply:
I think so, yes. Without this user may install zigpy either directly or as a dependency with Python 3.11 and scripts will fail. With this flag the installation won't succeed.

Reply:
I guess I should have been more clear. I haven’t actually tried to run this on 3.11 yet have you? I was curious if it does in fact have compatibility issues

Reply:
Oh, of course, I could have guessed that, sorry.

This is the error while running on Python 3.11:

```
ImportError while loading conftest '/Users/kblazewicz/git/github.com/zigpy/zigpy/tests/conftest.py'.
tests/conftest.py:7: in <module>
    import zigpy.application
zigpy/application.py:12: in <module>
    import zigpy.appdb
zigpy/appdb.py:14: in <module>
    import zigpy.backups
zigpy/backups.py:11: in <module>
    import zigpy.config as conf
zigpy/config/__init__.py:5: in <module>
    from zigpy.config.defaults import (
zigpy/config/defaults.py:1: in <module>
    import zigpy.types as t
zigpy/types/__init__.py:1: in <module>
    from .basic import *  # noqa: F401,F403
zigpy/types/basic.py:391: in <module>
    class bitmap2(bitmap_factory(uint2_t)):
zigpy/types/basic.py:305: in bitmap_factory
    class _NewEnum(int_type, enum.Flag):
zigpy/types/basic.py:309: in _NewEnum
    enum.IntFlag._create_pseudo_member_.__func__
../../../../.pyenv/versions/3.11.0/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/enum.py:767: in __getattr__
    raise AttributeError(name) from None
E   AttributeError: _create_pseudo_member_
```